samples/common/net_connect.c

```diff
@@ -73,8 +73,11 @@ void net_connect(void)
 {
 	struct net_if *iface = net_if_get_default();
 
-	LOG_INF("Waiting for interface to be up");
-	wait_for_iface_up(iface);
+	if (IS_ENABLED(CONFIG_GOLIOTH_SAMPLE_WIFI) &&
```

Comment:
We can combine this with the block below at line 82 to make it a little cleaner.

Reply:
Actually implemented check was broken. It was supposed to skip waiting for IFACE UP event for "WiFi && Dormant" interfaces, but still wait for IFACE UP for all others. Fixed that now. After the fix your comment about combining with block at 82 line does not apply anymore. But thanks for commenting about it, it allowed me to notice this bug. Tested with `mimxrt1060_evkb` and confirmed it works after the fix.